### PR TITLE
Disable Foresee during e2e tests and harden specific tests

### PR DESCRIPTION
**Pull request by jbalboni**
## Description
We've been seeing a lot more e2e test failures recently. This appears to be because the Foresee modal is popping up randomly in the middle of some tests. You can see this by running the tests locally with `vagovprod`.

The fix for this I chose is to create a `openUrl` wrapper for the normal `url` command. I did this because the script is triggered just being being in a script tag, so we can't conditionally turn it off at runtime. And we don't have a separate build type for e2e tests. I could have implemented this as a separate command and added to to every test, but that seemed more tedious than this approach.

## Testing done
Ran locally

## Acceptance criteria
- [x] Tests pass without failures (check the logs to look for failures that happened but went away after a retry)

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/platform/testing/e2e/nightwatch-commands/openUrl.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+exports.command = function openUrl(url, disableForesee = true) {
+  this.url(url);
+
+  if (disableForesee) {
+    this.execute(() => {
+      const css = document.createElement('style');
+      css.type = 'text/css';
+      css.innerHTML = '.__acs { display: none !important; }';
```
Comment (jbalboni) [inline src/platform/testing/e2e/nightwatch-commands/openUrl.js]: Foresee's API has a few methods of suppressing the modal, but none of them worked reliably for me. This does, though.

Comment (cvalarida) [reply src/platform/testing/e2e/nightwatch-commands/openUrl.js]: Clever.

**Review by cvalarida — APPROVED** [review]
Looking forward to less flaky e2e tests! :tada:

**Review by ncksllvn — APPROVED** [review]
Well done 🔥

**Pull request merged**